src/Api/Billing/Controllers/ProviderClientsController.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Bit.Api.Billing.Models.Requests;
+using Bit.Core;
+using Bit.Core.AdminConsole.Repositories;
+using Bit.Core.AdminConsole.Services;
+using Bit.Core.Billing.Commands;
+using Bit.Core.Context;
+using Bit.Core.Models.Business;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Billing.Controllers;
+
+[Route("providers/{providerId:guid}/clients")]
+public class ProviderClientsController(
```

Comment:
This file is a rename from `ProviderOrganizationsController` to emphasize it's specific to Consolidated Billing. The only net-new portion is the `CreateAsync` endpoint which will be used to create a new client organization under an MSP.